cpp/examples/basic/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
```

Comment:
Merge conflict? Nothing changed in this (and below) files.

Reply:
[Commented on this above](https://github.com/rapidsai/cudf/pull/13134#issuecomment-1507381612)